### ISSUE TITLE
Error:  "You have requested a non-existent service \"eight_points_guzzle.middleware.profile.mercury_api\"

### ISSUE THREAD

**Issue by omarysadek** [8p/EightPointsGuzzleBundle#235]
| Q                | A
| ---------------- | -----
| Symfony version  | 2.8.46
| Bundle version   | 7.4.0

**config.yml**

```
eight_points_guzzle:
    clients:
        myapi_api:
            base_url: "https://myurl"
            options:
                headers:
                    Content-Type: "application/json"
                curl:
                    sslversion: 1 # or !php/const:CURL_HTTP_VERSION_1_0 for symfony >= 3.2
```

**config_dev.yml**

```
eight_points_guzzle:
    logging: true
#    profiling: true
    clients:
        myapi_api:
            base_url: "https://dev-myurl"
            options:
                debug: true
                http_errors: true
```

```
$client   = $this->get('eight_points_guzzle.client.myapi_api');
```

> {"code":0,"message":"You have requested a non-existent service \"eight_points_guzzle.middleware.profile.myapi_api\". Did you mean one of these: \"eight_points_guzzle.client.myapi_api\", \"eight_points_guzzle.handler_stack.myapi_api\", \"eight_points_guzzle.middleware.event_dispatch.myapi_api\", \"eight_points_guzzle.middleware.log.myapi_api\", \"eight_points_guzzle.middleware.request_time.myapi_api\", \"eight_points_guzzle.middleware.symfony_log\"?"}

It works fine, when i downgrade the bundle to 7.3.1 version.

@florianpreusner i saw that you post a fix [here](https://github.com/8p/EightPointsGuzzleBundle/issues/165)

Am i missing something?

**Comment by gregurco** [comment]
@omarysadek if it Prod or Dev env? Could you provide your yaml configuration of guzzle bundle?

**Comment by gregurco** [comment]
Ans what happens if you will activate `profiling: true` ? Does the same error exist,

**Comment by omarysadek** [comment]
I tried with the minimal setup, here the code

```
eight_points_guzzle:
    clients:
        myapi_api:
            base_url: "https://myurl"
```

Still not working, the only way i found to make it working tis to downgrade the bundle, but i would like to use the lazy mode feature provided for 7.4

**Comment by gregurco** [comment]
ok, I will test it tomorrow with your version of symfony and I hope I will reproduce the error :slightly_smiling_face: I will write the result here.

**Comment by gregurco** [comment]
Fixed issue in PR #236
